bitnami/external-dns/templates/_helpers.tpl

```diff
@@ -163,6 +167,20 @@ Return the name of the Secret used to store the passwords
 {{- end -}}
 {{- end -}}
 
+{{- define "external-dns.alibabacloud-credentials" -}}
```

Comment:
From my point, it would be better to add this to a configmap instead of to a helper.

Reply:
Although it's called Alibaba Cloud configuration file, it actually stores access key id and secret, so I think it might be better to put it into a Secret instead of ConfigMap? 

Using the helper template is also the method used by the AWS and Azure provider for their respective credential files, so I was following the existing pattern here. What do you think?

Reply:
I agree a secret would be better.
Regarding the rest of the providers, maybe we should leave it in this way to maintain consistency and create a different PR to improve that, so you don't need to change it now.